src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -4001,9 +3997,14 @@ namespace EnergyPlus::ZoneEquipmentManager {
                 Node(ZoneNode).MassFlowRateMin = TotInletAirMassFlowRateMin;
                 Node(ZoneNode).MassFlowRateMinAvail = TotInletAirMassFlowRateMinAvail;
 
-                // Calculate standard return air flow rate using default method of inlets minus exhausts adjusted for "balanced" exhaust flow
-                StdTotalReturnMassFlow = TotInletAirMassFlowRate + ZoneMixingNetAirMassFlowRate -
-                                         (TotExhaustAirMassFlowRate - state.dataZoneEquip->ZoneEquipConfig(ZoneNum).ZoneExhBalanced);
+                if (state.dataHeatBal->ZoneAirMassFlow.ZoneFlowAdjustment != DataHeatBalance::NoAdjustReturnAndMixing) {
+                    // do not include adjusted for "balanced" exhaust flow
```

Comment:
So, if I'm following correctly, this change means that exhaust fan balancing will be ignored whenever ZoneAirMassFlowConservation is active? If that's what we want, then there should be a warning in the exhaust fan input processing that the balancing schedule will be ignored. But I'm wondering if a better solution would be to remove the balanced exhaust flow higher up from the mass conservation exhaust flow so it just disappears from the mass conservation balance up front.

Reply:
In this defect file the exhaust fan balancing terms leads to convergence problem and without the limit that I added on the zone return flow, the zone return flow rate could have been higher than the values we observed.

Reply:
OK. I will work to remove the balanced exhaust flow higher up from the mass conservation exhaust flow.

Reply:
Let's think this through before making more code changes. Maybe it's better to just warn that it's ignored when using ZoneAirMassFlowConservation. @EnergyArchmage @rraustad Any opinions on this?